sdk/anomalydetector/azure-ai-anomalydetector/swagger/README.md

```diff
@@ -1,25 +1,18 @@
 ## Generate autorest code
-``` yaml
-input-file:
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.1-preview.1/AnomalyDetector.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.1-preview.1/MultivariateAnomalyDetector.json
-use: '@autorest/java@4.0.24'
-java: true
+
+```yaml
+azure-arm: false
```

Comment:
please delete the incorrect autorest configure in readme https://github.com/Azure/azure-rest-api-specs/tree/feature/cognitiveservices/anomalydetector/mvad/specification/cognitiveservices/data-plane/AnomalyDetector#java